hikari/internal/typing_extensions.pyi

```diff
@@ -20,9 +20,13 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-"""Typehints for `hikari.internal.protocol`."""
+"""Typehints for `hikari.internal.typing_extensions`."""
 from typing import Protocol as __Protocol
+from typing import runtime_checkable as __runtime_checkable
 
-class Protocol(__Protocol): ...
+# fmt: off - Stops isort and black from fighting. Also, black makes this look weird
```

Comment:
How do they "fight" and what's wrong with "weird" if it follows the repo's formatting convention (unless your inference here is that it's a bug in which case it should probably be labled as that with a link to an issue for reference as to when the comments can just be removed)